app/src/main/java/com/codesoom/assignment/enums/HttpMethod.java

```diff
@@ -0,0 +1,23 @@
+package com.codesoom.assignment.enums;
+
+/**
+ * https://datatracker.ietf.org/doc/html/rfc7231#section-4.3
+ * */
```

Comment:
기왕 하는 김에 JavaDoc 태그를 사용해 봅시다. 이렇게 해 보세요. 그리고 IntelliJ 에서 HttpMethod 클래스 이름 위에 마우스를 올려보고 어떻게보이는지도 확인해 보세요.
```suggestion
/**
 * @see <a href="https://datatracker.ietf.org/doc/html/rfc7231#section-4.3">RFC 7231 section 4.3</a> 
 */
```

Reply:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/44446425/160235040-d5154b06-ae5b-4673-9b4a-d7330325f84d.png">

대박! 엄청신기하네요 👍🏾 이런게 있는 줄 몰랐습니다...!